src/JDNCalendarConversion.ts

```diff
@@ -68,20 +66,19 @@ export module JDNConvertibleConversionModule {
             month = calendarDate.month + 12;
         }
 
-        let c = 0;
-        if (year < 0) {
-            c = -0.75;
-        }
-        //
-        // we enforce Gregorian calendar for *all* dates (also prior to 1582)
-        //
+        let b = 0;
         let a = truncateDecimals(year/100.);
-        let b = 2 - a + truncateDecimals(a/4.);
+        let idate = year*10000 + month*100 + day;
```

Comment:
@gautschr Could you add a comment explaining why it is important to differentiate between dates before and after (or equal) 15.10.1582?

Reply:
maybe there could be a remark in the README about this as well.